_plugins/tags/content_block.rb

```diff
@@ -0,0 +1,18 @@
+module Jekyll
+  class ContentBlockTag < Liquid::Tag
+
+    def initialize(tag_name, text, tokens)
+      super
+      @text = text.strip
+    end
+
+    def render(context)
+      site = context.registers[:site]
+      if content_block = site.collections['content_blocks'].docs.detect{|b| b.data['slug'] == @text}
+        content_block.data.dig('content')
+      end
+    end
+  end
+end
+
+Liquid::Template.register_tag('content_block', Jekyll::ContentBlockTag)
```

Comment:
This should be moved into `jekyll-crds` but I'll handle that task later.

Reply:
This is now in the newly released [v2.3.1](https://github.com/crdschurch/jekyll-crds/releases/tag/2.3.1)